controllers/dependency_manager.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2020 The Flux CD contributors.
```

Comment:
```suggestion
Copyright 2020 The Flux authors
```

(We changed this yesterday)